mirrord/cli/src/main.rs

```diff
@@ -348,7 +348,7 @@ async fn exec(args: &ExecArgs, watch: drain::Watch) -> Result<()> {
     execution_result
 }
 
-async fn list_pods(layer_config: &LayerConfig, args: &ListTargetArgs) -> Result<Vec<String>> {
+async fn list_targets(layer_config: &LayerConfig, args: &ListTargetArgs) -> Result<Vec<String>> {
```

Comment:
```suggestion
/// Lists targets based on whether or not the operator has been enabled in `layer_config`.
/// If the operator is enabled (and we can reach it), then we list [`Seeker::all`] targets, otherwise we list [`Seeker::all_open_source`] only.
async fn list_targets(layer_config: &LayerConfig, args: &ListTargetArgs) -> Result<Vec<String>> {
```

Or something along those lines.